webapp/src/Service/CheckConfigService.php

```diff
@@ -316,55 +308,58 @@ public function checkScriptFilesizevsMemoryLimit(): array
              $result = 'O';
         }
         $this->stopwatch->stop(__FUNCTION__);
-        return ['caption' => 'Compile file size vs. memory limit',
-                'result' => $result,
-                'desc' => 'If the script filesize limit is lower than the memory limit, then ' .
-                'compilation of sources that statically allocate memory may fail. We ' .
-                'recommend to include a margin to be on the safe side. The current ' .
-                '"script_filesize_limit" = ' . $this->config->get('script_filesize_limit') . ' ' .
-                'while "memory_limit" = ' . $this->config->get('memory_limit') . '.'
-            ];
+        return new ConfigCheckItem(
+            caption: 'Compile file size vs. memory limit',
+            result: $result,
+            desc: 'If the script filesize limit is lower than the memory limit, then ' .
+            'compilation of sources that statically allocate memory may fail. We ' .
+            'recommend to include a margin to be on the safe side. The current ' .
+            '"script_filesize_limit" = ' . $this->config->get('script_filesize_limit') . ' ' .
+            'while "memory_limit" = ' . $this->config->get('memory_limit') . '.'
+        );
```

Comment:
```suggestion
        return new ConfigCheckItem(
            caption: 'Compile file size vs. memory limit',
            result: $result,
            desc: 'If the script filesize limit is lower than the memory limit, then ' .
                'compilation of sources that statically allocate memory may fail. We ' .
                'recommend to include a margin to be on the safe side. The current ' .
                '"script_filesize_limit" = ' . $this->config->get('script_filesize_limit') . ' ' .
                'while "memory_limit" = ' . $this->config->get('memory_limit') . '.'
        );
```

Reply:
Does PSR12 say something about this? My .editorconfig, code style in PHPStorm as well as phpcs doesn't complain about this and PHPStorm even changes it back to be without indent.

Reply:
There MUST NOT be a hard limit on line length; the soft limit MUST be 120 characters; lines SHOULD be 80 characters or less. (PSR2), PSR12 doesn't have a new opinion.

I just find it ugly to look at but lets leave it.